app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -37,37 +37,84 @@ params:
     - name: handle_unknown
       default: "`false`"
       required: false
-      description: Allow transform to apply to unmatched route (404) responses. Should not be enabled on more than one plugin configuration.
+      description: Allow transform to apply to unmatched Service, Route, or Workspace (404) responses.
     - name: handle_unexpected
       default: "`false`"
       required: false
-      description: Allow transform to apply to unexpected request (400) responses. Should not be enabled on more than one plugin configuration.
-    - name: handle_admin
-      default: "`false`"
-      required: false
-      description: Allow transform to apply to Admin API responses. Should not be enabled on more than one plugin configuration.
+      description: Allow transform to apply to unexpected request (400) responses.
 
 ---
 
-## Transforming 404 and 400 responses
-
-By default, the exit transformer is only applied to requests that match its
-criteria (its route, service, or consumer matching configuration) or
-globally within a workspace. However, requests that result in 400 or 404
-responses neither match any criteria nor fall within any specific workspace,
-and standard plugin criteria will never match them. Users can designate exit
-transformer configurations that _do_ handle these responses by enabling the
-`handle_unknown` (404) and `handle_unexpected` (400) settings. These should
-only be enabled on a single plugin configuration.
-
-`handle_admin` allows the exit transformer to apply to Admin API responses.
-Users should only modify headers only applying functions to Admin API
-responses, as modifying the body or status will interfere with Kong Manager's
-ability to communicate with the Admin API.
-
-## Function syntax
-
-The exit transformer expects a configuration function to be Lua code that returns
+## Transforming 4xx and 5xx Responses
+
+By default, the Exit Transformer is only applied to requests that match its
+criteria (its Route or Service matching configuration) or globally within a Workspace.
+
+### Handling Unmatched 400 and 404 Responses
+
+Requests that result in 400 or 404 responses neither match any criteria nor fall
+within any specific Workspace, and standard plugin criteria will never match those
+responses. You can designate Exit Transformer configurations that _do_ handle these
+responses by enabling the `handle_unexpected` (400) and `handle_unknown` (404) settings.
+The `handle_unknown` parameter should only be enabled on a single plugin configuration.
+The `handle_unexpected` parameter can be enabled on as many plugin configurations
```

Comment:
Suggest breaking these two lines into bullet points for better readability, and to break up this block of text.

Reply:
Done